### PR TITLE
[REF] web: graph and pivot: rework view props

### PR DESCRIPTION
**Pull request by aab-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by aab-odoo** [comment]
robodoo rebase-ff

File: addons/web/static/src/views/graph/graph_view.js [hunk 1]
```diff
@@ -17,37 +17,31 @@ const viewRegistry = registry.category("views");
...
-            for (const key of KEYS) {
-                modelParams[key] = key in archInfo ? archInfo[key] : this.props[key];
-            }
+            modelParams = {
+                additionalMeasures: this.props.additionalMeasures,
+                disableLinking: Boolean(archInfo.disableLinking),
+                displayScaleLabels: this.props.displayScaleLabels,
+                fieldAttrs: archInfo.fieldAttrs,
+                fields: this.props.fields,
+                groupBy: archInfo.groupBy,
```
Comment (Polymorphe57) [inline addons/web/static/src/views/graph/graph_view.js:37]: maybe avoid to set default in arch processor (du coup) and do what you do for stacked here?

File: addons/web/static/tests/views/graph_view_tests.js [hunk 2]
```diff
@@ -2864,6 +2865,7 @@ QUnit.module("Views", (hooks) => {
     QUnit.test('graph view with attribute disable_linking="1"', async function (assert) {
         assert.expect(4);
 
+        serviceRegistry.add("localization", makeFakeLocalizationService());
```
Comment (Polymorphe57) [inline addons/web/static/tests/views/graph_view_tests.js:2868]: I don't understand why you need this in this commit

File: addons/web/static/tests/views/graph_view_tests.js [hunk 3]
```diff
@@ -3326,7 +3326,7 @@ QUnit.module("Views", (hooks) => {
                     </graph>`,
                 searchViewArch: `
                     <search>
-                        <filter name="my_filter" string="My Filter" domain="[('id', '&lt;', 2)]"/>
+                        <filter name="my_filter" string="My Filter" domain="[('id', '&lt;', 6)]"/>
```
Comment (Polymorphe57) [inline addons/web/static/tests/views/graph_view_tests.js:3329]: why do you change that?

File: addons/web/static/tests/views/graph_view_tests.js (hunk 2 above)
Comment (aab-odoo) [reply addons/web/static/tests/views/graph_view_tests.js:2868]: we use _t to default the title. In this test, we mock the action service, so we don't have localization automatically

File: addons/web/static/tests/views/graph_view_tests.js (hunk 3 above)
Comment (aab-odoo) [reply addons/web/static/tests/views/graph_view_tests.js:3329]: to have meaningful data

**Comment by ged-odoo** [comment]
robodoo r+

**Pull request closed without merging**